features/bundle/package.json

```diff
@@ -29,19 +24,14 @@
       "artifactPath": "dist"
     },
     {
-      "archiveName": "polyExplorer",
-      "moduleName": "poly-explorer-feature",
+      "archiveName": "facebookImport",
+      "moduleName": "facebook-import-feature",
       "artifactPath": "dist"
     },
     {
       "archiveName": "lexicon",
       "moduleName": "lexicon-feature",
       "artifactPath": "public"
-    },
-    {
```

Comment:
Probably a good call to hide this...

Reply:
Yeah, they were only meant to be shown in the internal builds. But since we use the exact same builds for internal builds and releases (which is great), they had to (temporarily) go now. I do dream of having some settings that allow users to enable experimental or testing related features, maybe behind some cute easter egg, but not today :D